addons/website/static/src/client_actions/configurator/configurator.js

```diff
@@ -5,7 +5,7 @@ import { delay } from "@web/core/utils/concurrency";
 import { getDataURLFromFile } from "@web/core/utils/urls";
```

Comment:
Commit message:
4th commit:
- > the colors of the logo was not detected

  -> were not detected
- > the possibility to detect the color
  
  -> the colors